test/utils.go

```diff
@@ -95,3 +97,12 @@ func IsTLSExist() bool {
 	}
 	return true
 }
+
+func inspectFilter(name, filter string) (string, error) {
```

Comment:
This encapsulated function looks so good to me. 😄 🌹 🍻

Reply:
done.